dashboard/lib/build_dashboard_page.dart

```diff
@@ -16,6 +16,7 @@ import 'widgets/error_brook_watcher.dart';
 import 'widgets/filter_property_sheet.dart';
 import 'widgets/task_box.dart';
 import 'widgets/task_grid.dart';
+import 'package:icon_plugin/icon_plugin.dart';
```

Comment:
re the failing test:

```
The following MissingPluginException was thrown running a test (but after the test had completed):
```

You'll need to update the plugin so that if there's no implementation for the current platform, it no-ops all calls.

There's docs in https://docs.flutter.dev/development/packages-and-plugins/developing-packages#plugin and it's generally referred to as Flutter's federated plugin structure. You'll define a generic interface that then all the platforms implement.